app/javascript/components/patient/household/HouseholdMemberTable.js

```diff
@@ -259,14 +259,14 @@ class HouseholdMemberTable extends React.Component {
 
     return (
       <React.Fragment>
-        {rowData.head_of_household && <BadgeHoH patientId={rowData.id.toString()} customClass={'float-right ml-1'} location={'right'} />}
         <a
           id={rowData.head_of_household ? 'dependent-hoh-link' : null}
           href={patientHref(rowData.id, this.props.workflow)}
           target={this.props.isSelectable ? '_blank' : '_self'}
           rel="noreferrer">
           {monitoreeName}
         </a>
+        {rowData.head_of_household && <BadgeHoH patientId={rowData.id.toString()} customClass={'float-right ml-1'} location={'right'} />}
```

Comment:
semantically this comes after the name, so why was it ever placed before? This doesnt change the visuals at all, just the semantics.